[IMP] pos: update fiscal positions page (#3876)
odoo/documentation: content/applications/sales/point_of_sale/restaurant/fiscal_position.rst

=================================
Flexible taxes (fiscal positions)
=================================

When running a business, you may need to apply different taxes and record transactions on various
accounts based on the location and type of business of your customers and providers.

The **fiscal positions** feature enables you to establish rules that automatically adjust
the taxes and accounts used for each transaction.

.. seealso::
   - :doc:`../../../finance/accounting/taxation/taxes/fiscal_positions`
   - :doc:`../../../finance/accounting/taxation/taxes/taxes`

Configuration
=============

To enable the feature, go to :menuselection:`Point of Sale --> Configuration --> Settings`, scroll
down to the :guilabel:`Accounting` section, and enable :guilabel:`Flexible Taxes`.

> Then, set a default fiscal position that should be applied to all sales in POS in the

Reviewer: "to all sales in POS" sounds like this setting is applied to the whole app (which would be "Point of Sale" instead, then)

It isn't clear here that this action is applied to the current pos. I know it's the case for all settings, but it wouldn't hurt to add "select the pos" at the beginning.

:guilabel:`Default` field and all the other allowed fiscal positions to choose from in the
:guilabel:`Allowed` field.

.. image:: fiscal_position/flexible-taxes-setting.png
   :align: center

> According to the fiscal localization package activated, several fiscal positions are preconfigured

Reviewer: I suggest adding a link to "fiscal localization package"

> and can be set and used in POS. However, to create new ones, :ref:`follow the related documentation
> <fiscal_positions/mapping>`.

Reviewer: Avoid "click here" links and try to put meaningful text on it. For example:
```suggestion
and can be set and used in POS. You can also :ref:`create new fiscal positions
<fiscal_positions/mapping>`.
```

.. note::

>    If you do not set a fiscal position, the products are sold based on the customer taxes assessed
>    on the product form.

Reviewer: ```suggestion
   If you do not set a fiscal position, the tax remains as defined
   on the product form.
```

Reviewer: or "set" instead of "defined" since you seem to use that wording in line 48.

Use fiscal positions
====================

Open a :ref:`POS session <pos/start-session>` to use one of the allowed fiscal positions. Then,

> click the :guilabel:`Tax` button next to the **book-shaped** icon and select from the list of

Reviewer: You need to select something (to put before "from the list")
